diesel_cli/src/main.rs

```diff
@@ -122,6 +139,19 @@ fn run_migration_command(matches: &ArgMatches) {
             println!("Creating {}", migration_dir_relative.join("down.sql").display());
             fs::File::create(down_path).unwrap();
         }
+        ("manage-timestamps", Some(args)) => {
+            let table_name = args.value_of("TABLE_NAME").unwrap();
+            let connection = connection(&database_url(args));
+            match connection.manage_updated_at_for_table(&table_name) {
```

Comment:
Could this be written as `connection.manage_updated_at_for_table(&table_name).unwrap_or_else(handle_error)`?